### PR TITLE
Implement dark mode & basic toggle

### PR DESCRIPTION
**Pull request by GabrielChouinardLetourneau**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Feature

## Description

Implementation of the Dark mode

- Changes in Brushes and overall cohesion of the themes
- Add basic toggle in settings to change the theme (doesn't save in any way atm/not storing the current theme of the user)

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs [hunk 1]
```diff
@@ -10,13 +10,30 @@
 using Chinook.StackNavigation;
 using MessageDialogService;
 using Microsoft.Extensions.Localization;
+using Windows.UI.Xaml;
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs:13]: Don't manipulate the view from the presentation layer.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs [hunk 2]
```diff
@@ -10,13 +10,30 @@
...
 namespace ApplicationTemplate.Presentation
 {
 	public partial class SettingsPageViewModel : ViewModel
 	{
+
+		public SettingsPageViewModel()
+		{
+			AddDisposable(this.GetProperty(x => x.IsThemeChangeEnabled)
+				.Observe()
+				.SelectManyDisposePrevious((e, ct) => OnThemeSwitchButtonChange())
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs:26]: Use commands when you want to implement commands.
You should very rarely use `SelectManyDisposePrevious`.

Comment (jeanplevesque) [reply src/app/ApplicationTemplate.Shared/Presentation/Settings/SettingsPageViewModel.cs:26]: FYI you can invoke a command from a ToggleSwitch like this.

```xaml
<ToggleSwitch IsOn="{Binding SomeBoolean, Mode=TwoWay}">
	<interactivity:Interaction.Behaviors>
		<core:EventTriggerBehavior EventName="Toggled">
			<core:InvokeCommandAction Command="{Binding SomeCommand}" />
		</core:EventTriggerBehavior>
	</interactivity:Interaction.Behaviors>
</ToggleSwitch>
```

File: src/app/ApplicationTemplate.UWP/Views/Content/Settings/SettingsPage.xaml [hunk 3]
```diff
@@ -6,6 +6,8 @@
 	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  xmlns:ue="using:Nventive.View.Extensions"
 	  xmlns:dl="using:Chinook.DataLoader"
+	  xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/Settings/SettingsPage.xaml:9]: needed?

**Review by carlh98 — APPROVED** [review]

**Pull request merged**